lib/src/private/mli_prv_dsp.h

```diff
@@ -569,26 +569,6 @@ static inline void __attribute__ ((always_inline)) mli_prv_load_mac_vec4(acc_T *
     mli_prv_load_mac_vec2(accu, in, k);
 }
 
-static inline void __attribute__ ((always_inline)) mli_prv_load_mac_vec4(
```

Comment:
for 8bit weights there is a benefit to use _dmachbl and _dmachbm because only a single 32bit load is needed to load four 8bit values.
did you measure the performance impact of this change?

Reply:
Yes, I made measurements on em5d. Performance improved.
Also, function mli_prv_load_mac_vec2 already have overloads with _dmachbl and _dmachbm.

Reply:
mli_prv_load_mac_vec2() only uses _dmachbl. when both _dmachbl and _dmachbm are used together, two load operations could be combined into one load operation.